frontend/public/components/pod.jsx

```diff
@@ -139,6 +139,28 @@ const Volume = ({pod, volume}) => {
   </div>;
 };
 
+const ContainerTable = ({heading, containers, pod}) => {
```

Comment:
```jsx
const ContainerTable = ({heading, containers, pod}) => <div className="co-m-pane__body">
  ...
</div>;
```

Reply:
Ah, yup, that doesn't need the curlies anymore.